codegen/allapigen.py

```diff
@@ -1,9 +1,34 @@
+import argparse
+import os
+
 import datamodelgen
 import print_fluent_version
 import settingsgen
 import tuigen
 
+from ansys.fluent.core.launcher.launcher import FLUENT_VERSION, set_ansys_version
+
 if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description="Generate python code from Fluent APIs"
+    )
+    parser.add_argument(
+        "--ansys-version",
```

Comment:
new command line arguments.   to be able to run icing (assuming later on i point to the proper fluent connection) i would then need to 

`python codegen/allapigen.py --ansys-version=23.1`